src/plugins/orangehrmPerformancePlugin/Exception/PerformanceTrackerServiceException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace OrangeHRM\Performance\Exception;
+
+use Exception;
+
+class PerformanceTrackerServiceException extends Exception
+{
+    /**
+     * @return static
+     */
+    public static function cannotEditEmployeeWithLogs(): self
+    {
+        return new self("The employee cannot be updated since the performance tracker already contains review logs.");
```

Comment:
```suggestion
        return new self("The employee cannot be updated since the performance tracker already contains review logs");
```
:thinking: